packages/iris-grid/src/IrisGridPartitionedTableModel.ts

```diff
@@ -0,0 +1,442 @@
+    value: unknown,
+    columnType: string,
+    columnName?: string | undefined
+  ): string {
+    return this.model.displayString(value, columnType, columnName);
+  }
+
+  get sort(): readonly Sort[] {
+    return this.model.sort;
+  }
+
+  set sort(value: readonly Sort[]) {
+    this.model.sort = value;
+  }
+
+  get customColumns(): readonly ColumnName[] {
+    return this.model.customColumns;
+  }
+
+  set customColumns(customColumns: readonly ColumnName[]) {
+    this.model.customColumns = customColumns;
+  }
+
+  get formatColumns(): readonly CustomColumn[] {
+    return this.model.formatColumns;
+  }
+
+  set formatColumns(formatColumns: readonly CustomColumn[]) {
+    this.model.formatColumns = formatColumns;
+  }
+
+  updateFrozenColumns(columns: ColumnName[]): void {
+    this.model.updateFrozenColumns(columns);
+  }
+
+  get rollupConfig(): RollupConfig | null {
+    return this.model.rollupConfig;
+  }
+
+  set rollupConfig(rollupConfig: RollupConfig | null) {
+    this.model.rollupConfig = rollupConfig;
+  }
+
+  get totalsConfig(): UITotalsTableConfig | null {
+    return this.model.totalsConfig;
+  }
+
+  set totalsConfig(totalsConfig: UITotalsTableConfig | null) {
+    this.model.totalsConfig = totalsConfig;
+  }
+
+  get layoutHints(): LayoutHints | null {
+    return this.model.layoutHints;
+  }
+
+  get frontColumns(): readonly ColumnName[] {
+    return this.model.frontColumns;
+  }
+
+  get backColumns(): readonly ColumnName[] {
+    return this.model.backColumns;
+  }
+
+  get frozenColumns(): readonly ColumnName[] {
+    return this.model.frozenColumns;
+  }
+
+  isColumnFrozen(index: ModelIndex): boolean {
+    return this.model.isColumnFrozen(index);
+  }
+
+  isColumnSortable(index: ModelIndex): boolean {
+    return this.model.isColumnSortable(index);
+  }
+
+  get isFilterRequired(): boolean {
+    return this.model.isFilterRequired;
+  }
+
+  get isPartitionRequired(): boolean {
+    return true;
+  }
+
+  get isReversible(): boolean {
+    return this.model.isReversible;
+  }
+
+  export(): Promise<Table> {
+    return this.model.export();
+  }
+
+  get isColumnStatisticsAvailable(): boolean {
+    return this.model.isColumnStatisticsAvailable;
+  }
+
+  get description(): string {
+    return this.model.description;
+  }
+
+  columnStatistics(column: Column): Promise<ColumnStatistics> {
+    return this.model.columnStatistics(column);
+  }
+
+  get isCustomColumnsAvailable(): boolean {
+    return this.model.isCustomColumnsAvailable;
+  }
+
+  get isFormatColumnsAvailable(): boolean {
+    return this.model.isFormatColumnsAvailable;
+  }
+
+  get isExportAvailable(): boolean {
+    return this.model.isExportAvailable;
+  }
+
+  get isValuesTableAvailable(): boolean {
+    return this.model.isValuesTableAvailable;
+  }
+
+  get isChartBuilderAvailable(): boolean {
+    return this.model.isChartBuilderAvailable;
+  }
+
+  get isRollupAvailable(): boolean {
+    return this.model.isRollupAvailable;
```

Comment:
Hmm we've got a problem right now if you try and set a rollup or set a select distinct on a partitioned table then it throws an error. I think you should just return `false` in these cases.